Variant.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 11:56:17 2016
+
+@author: Mina Naghshnejad
+"""
+
+class Variant():
```

Comment:
We may not need a Variant object at this point ( @detiber ? ) but, using the example WorkflowConfig from #4, an abstract Variant object would have an identifier, a name, a list of workflows and a list of roles. Variant should also inherit from `object`. This object should be free of anything openshift specific since it may be used for other purposes.